mbed_devices/_internal/mbed_tools/list_connected_devices.py

```diff
@@ -17,9 +17,22 @@
 @click.option(
     "--format", type=click.Choice(["table", "json"]), default="table", show_default=True, help="Set output format."
 )
-def list_connected_devices(format: str) -> None:
+@click.option(
+    "--show-all",
+    "-a",
+    is_flag=True,
+    default=False,
+    help="Show all connected devices, even those which are not Mbed Targets.",
+)
+def list_connected_devices(format: str, show_all: str) -> None:
     """Prints connected devices."""
-    devices = _sort_devices_by_name(get_connected_devices())
+    identified_devices, unidentified_devices = get_connected_devices()
+
+    if show_all:
```

Comment:
I'm confused by this. `show_all` is a `str` yet we only check if it evaluates to `True`? Why not just use a `bool` flag? Maybe I'm missing something in the diff.

Reply:
Good spot, it should be a bool as it is in the click options.